live_chat_sale_reference/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This modules does the following:
+
+* Add Sales order reference link at chat when the sale number is used in chat.
```

Comment:
This module does the following:

* Replace sale order name to a reference link to the sales order in the internal chat.